utils/rate_limiter.go

```diff
@@ -1,77 +0,0 @@
-// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
please do not do this in one PR - replacing RateLimiter should be its own PR.

Reply:
OK will do